### ISSUE TITLE
Pressing Ctrl+? does not open the Keyboard Shortcuts window as expected

### ISSUE THREAD

**Issue by slook** [nicotine-plus/nicotine-plus#1484]
Nicotine+ version: 3.1.1  •  GTK 3.24.21 (Flatpak)
Nicotine+ version: 3.0.6 (PIP3)
Operating System/Distribution: MX Linux 19 Debian 9 (Stretch) XFCE 4.12
Keyboard localization: UK (English)

### Describe the bug

Pressing `Ctrl+?` does not open the Keyboard Shortcuts window as expected.

It is not obvious nor intuitive that `Shift+Ctrl+?` is infact the correct shortcut combination to press.

### Expected behavior

Pressing `Ctrl+?` (without Shift) should open the Keyboard Shortcuts window, as stated in that window under the "Miscellaneous" category.

To resolve this bug, the information provided should be changed to `Shift+Ctrl+?` because pressing `Shift` is required to type a `?` character on most common keyboard layouts. In the UK, the `?` character shares a key with the forward slash `/` character. This item would then look nicest at the top of the Miscellaneous category, above Connect. 

Pressing `Ctrl+/` (without Shift) seems to have the same function as `Ctrl+A` when a list object has the focus. If this can be overridden then `Ctrl+/` would be useful as an additional alternative shortcut for the Keyboard Shortcuts window shortcut.

### Ideal behavior

The `F1` key has no function currently, some help would be nice whilst N+ doesn't have any help function, so perhaps this key could be assigned to this function instead of or in addition to `Shift+Ctrl+?` ... however the window in question does not really give any clues about how the interface is operated, so maybe F1 isn't a good idea for this in the long-run... I suppose ultimately in the far future the Keyboard Shortcuts window might ideally allow hotkey customization (in which case I guess it would then be in the Preferences window instead of being it's own dialog box).

Even more elaborate idea for future enhancement(!)... perhaps the Keyboard Shortcuts window could be re-purposed to be a Quick Help Guide of some sort, perhaps like the about box with some top tabs for three or four sections; to briefly describe what each main window tab does; how to login to a server go online or away; how to search and download a file; how to join a room and use chat; keyboard shortcuts. Then, only if the window contained useful guidance, the `F1` key would make sense.

It would be an added bonus if the same shortcut key(s) also closed the Keyboard Shortcuts help window if it is open, if possible, in addition to `Esc` of course.

TL;DR: To resolve this bug, the information provided should be changed to `Shift+Ctrl+?` because pressing `Shift` is required to type a `?` character on most common keyboard layouts.

### Steps to reproduce the bug

0. Disconnect mouse (optional step)
1. Press `Ctrl+?` (without Shift)
2. Watch when nothing happens :-(
3. Press `F10` > `Alt+H` > `Alt+K` instead,
4. Read the displayed information and close window,
6. Press `Ctrl+?` again as per given information,
7. Watch when nothing happens :-(
8. Be frustrated!

### Additional context

This is an accessibility problem, see link:
[Web Content Accessibility Guidelines to Non-Web Information and Communications Technologies (WCAG2ICT) - Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/#keyboard-accessible)

Guideline 2.1 (Keyboard Accessible) requires that **"All functionality of the content is operable through a keyboard interface"**.

It is important to consider that a novice keyboard-only user may well be unaware that F10 opens the main menu, in which case they would be entirely prevented from operating the client's interface. Such a user might first press F1 (no function), then Ctrl+? (no function), then (also because switching main window tabs requires a mouse) they would abandon the application.

There could be many reasons why a user may be unable to use a mouse or other pointing device... disability, visual impairment, cramped workspace (such as a DJ console), bright flashing lights at a party and/or being too drunk!

Please note there are also other accessibility problems (mainly tab-navigation imo) that prevents compliance with WCAG2ICT Guideline 2.1 and mouse-less operation, see issue #1010
